### ISSUE TITLE
display of encrypted file

### ISSUE THREAD

**Issue by aleks-cortijo** [inwe-boku/lecture-scientific-programming#12]
i first generated and pushed the file in my general homework-scientific-computing repository and not into the homework2-conda-python folder. 
after i realized that, i did the all the encryption etc. again, now in the correct folder and pushed again.

looking at the file on github now the the .txt file in the homework2-conda-python folder is immediately displayed, whereas the one in homework-sci displays the "view raw" hyperlink.

Any chance to change the one from immediately displayed to "view raw"? What went wrong?

**Comment by lumbric** [comment]
I guess you are talking about the [preview here](https://github.com/aleks-cortijo/homework-scientific-computing/blob/master/homework02-conda-python/group-member-aleks-cortijo.txt) and the [missing preview here](https://github.com/aleks-cortijo/homework-scientific-computing/blob/master/group-member-aleks-cortijo.txt). This is indeed a bit surprising, Github seems to behave a bit weird here - could be a Github bug. The only plausible explanation I could come up with: the encrypted files are different. Mabye one of the files contains a character which cannot be interpreted as valid UTF-8 character and therefore Github assumes it is a binary file. So I guess somehow Github's heuristic which is responsible for determining if the file is binary or human-readable fails in this case.

I also don't have an explanation why the encrypted files are different. Both can be decrypted and contain the same clear text message AFAICS. I did not look into details of the encryption mechanism we chose, but I guess it should be deterministic.

Anyway for the purpose of the exercise, it doesn't matter. The preview does not make sense for the encrypted files anyway, we need to download them to decrypt them.

This did not really answer your questions, but it is the best I can come up with at the moment. Feel free to close the ticket, if you don't have to add anything else.
